replica/compaction_group.hh

```diff
@@ -14,13 +14,15 @@
 #include "compaction/compaction_backlog_manager.hh"
 #include "compaction/compaction_strategy_state.hh"
 #include "sstables/sstable_set.hh"
-#include "compaction/compaction_fwd.hh"
+#include "utils/chunked_vector.hh"
+#include <boost/intrusive/list.hpp>
 
 #pragma once
 
 namespace replica {
 
 using enable_backlog_tracker = bool_class<class enable_backlog_tracker_tag>;
+using namespace boost::intrusive;
```

Comment:
You just removed a `using namespace` declaration, don't add a new one.

Reply:
Fair enough.